requirements/edx-sandbox/base.in

```diff
@@ -22,8 +22,8 @@ pyparsing==2.2.0                    # Python Parsing module
 random2                             # Implementation of random module that works identically under Python 2 and 3
 scipy==0.14.0                       # Math, science, and engineering library
 sympy==0.7.1                        # Symbolic math library
+chem
```

Comment:
Could you also place this in the right place to get this file sorted? the same for base.in